### PR TITLE
Remove fuzzy finder cache storage.

### PR DESCRIPTION
**Pull request by olafurpg**
Previously, it was slow to download files in larger repositories so the
fuzzy finder used the cache storage to speed up subsequent loads. This
commit removes that caching because

- downloading files has gotten faster https://github.com/sourcegraph/sourcegraph/pull/21422
- filenames are already cached by the SPA as the user navigates between
  files. Files only need to be re-downloaded when refreshing the
  browser. The original implementation used raw `<a href=` links, which trigger
  a cold refresh.

### CODE REVIEW

**Comment by olafurpg** [comment]
The CI failure looks unrelated
```
No preset version installed for command node
Please install a version by running one of the following:

or add one of the following versions in your config file at /buildkite/builds/buildkite-agent-65c8b757c8-tphm4-1/sourcegraph/sourcegraph/.tool-versions

nodejs 13.12.0
nodejs 12.16.1
```

**Review by eseliger — APPROVED** [review]

File: .tool-versions [hunk 1]
```diff
@@ -5,3 +5,4 @@ shfmt 3.2.0
 shellcheck 0.7.1
 kubectl 1.17.3
 github-cli 1.8.0
+nodejs 14.15.4
```
Comment (olafurpg) [inline .tool-versions:8]: @eseliger this unrelated change seems to be necessary to get the CI green in this PR. Is it OK to add this here or should it be a separate PR?

Comment (eseliger) [reply .tool-versions:8]: We have a separate file `.nvmrc` which is also supported by asdf, but also renovate which updates this version for us from time to time, so we should already be on that version. I don't see a build failure on the previous commit, why is this required?

Comment (olafurpg) [reply .tool-versions]: I reverted this change and the CI is green now. Yesterday, it was failing with this error message
```
    No preset version installed for command node
    Please install a version by running one of the following:

    or add one of the following versions in your config file at /buildkite/builds/buildkite-agent-65c8b757c8-tphm4-1/sourcegraph/sourcegraph/.tool-versions

    nodejs 13.12.0
    nodejs 12.16.1
```

**Comment by olafurpg** [comment]
>  codecov/patch — 0.00% of diff hit (target 5.00%) 

Is it OK to ignore this failing CI check? We're tracking followup improvements to the fuzzy finder in https://github.com/sourcegraph/sourcegraph/issues/21201 and one of the items on the list is to add storybook tests

File: .tool-versions (hunk 1 above)
Comment (eseliger) [reply .tool-versions]: Oh I saw this a couple of times.. for some reason asdf doesn't work properly some times (?)

**Comment by eseliger** [comment]
Yes! It is just a friendly reminder that the changed code had no tests :)

**Comment by olafurpg** [comment]
Good to know! It's a healthy reminder indeed :)

File: .tool-versions (hunk 1 above)
Comment (olafurpg) [reply .tool-versions]: It's good to know this error may happen sporadically.

**Pull request merged**